amazon-dynamodb-quickstart/pom.xml

```diff
@@ -12,12 +12,14 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+        <quarkus-plugin.version>${quarkus.platform.version}</quarkus-plugin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <quarkus-amazon-services.version>1.0.4</quarkus-amazon-services.version>
         <awssdk.testcontainers.version>1.12.57</awssdk.testcontainers.version>
+        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+        <failsafe-plugin.version>${surefire-plugin.version}</failsafe-plugin.version>
```

Comment:
How similar is it to what we have when we create a project with the tooling? Because I think we should try to align things with that as much as we can.

Reply:
The values of all of these params are the same, but by this implementation, we are able to overwrite some parameters by system properties. 

for example,

`mvn clean install -Dquarkus.platform.group-id=com.redhat.quarkus.platform -Dquarkus.platform.version=2.2.5.Final-redhat-00007  -Dquarkus.platform.artifact-id=quarkus-bom`